platform/android/export/export.cpp

```diff
@@ -1341,6 +1356,42 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 			return str;
 		}
 	}
+
+	// Creates strings.xml files inside the gradle project for different locales.
+	void _create_project_name_strings_files(const Ref<EditorExportPreset> &p_preset) {
```

Comment:
For standalone methods like this one, would it be cleaner to put them in a separate file, i.e. `gradle_export_util.h`?

Reply:
Sounds like a good idea!